2021-09-06 11:29:52,732 INFO    MainThread:22428 [wandb_setup.py:_flush():69] setting env: {}
2021-09-06 11:29:52,733 INFO    MainThread:22428 [wandb_init.py:_log_setup():342] Logging user logs to C:\Programming\rl_cooperation\Pygame_RL_cooperation\SpaceCannons\wandb\offline-run-20210906_112952-2j7f7lki\logs\debug.log
2021-09-06 11:29:52,733 INFO    MainThread:22428 [wandb_init.py:_log_setup():343] Logging internal logs to C:\Programming\rl_cooperation\Pygame_RL_cooperation\SpaceCannons\wandb\offline-run-20210906_112952-2j7f7lki\logs\debug-internal.log
2021-09-06 11:29:52,733 INFO    MainThread:22428 [wandb_init.py:init():375] calling init triggers
2021-09-06 11:29:52,733 INFO    MainThread:22428 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-06 11:29:52,734 INFO    MainThread:22428 [wandb_init.py:init():424] starting backend
2021-09-06 11:29:52,734 INFO    MainThread:22428 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-06 11:29:52,741 INFO    MainThread:22428 [backend.py:ensure_launched():135] starting backend process...
2021-09-06 11:29:52,874 INFO    MainThread:22428 [backend.py:ensure_launched():139] started backend process with pid: 26804
2021-09-06 11:29:52,875 INFO    MainThread:22428 [wandb_init.py:init():429] backend started and connected
2021-09-06 11:29:52,876 INFO    MainThread:22428 [wandb_init.py:init():477] updated telemetry
2021-09-06 11:29:52,878 INFO    MainThread:22428 [wandb_init.py:init():540] starting run threads in backend
2021-09-06 11:29:54,881 INFO    MainThread:22428 [wandb_run.py:_console_start():1601] atexit reg
2021-09-06 11:29:54,882 INFO    MainThread:22428 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-06 11:29:54,883 INFO    MainThread:22428 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-06 11:29:54,885 INFO    MainThread:22428 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-06 11:29:54,885 INFO    MainThread:22428 [wandb_init.py:init():565] run started, returning control to user process
2021-09-06 11:29:54,892 INFO    MainThread:22428 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-09-06 11:29:54,892 INFO    MainThread:22428 [wandb_run.py:_restore():1543] restore
2021-09-06 11:29:55,921 INFO    MainThread:22428 [wandb_run.py:_wait_for_finish():1693] got exit ret: 
2021-09-06 11:29:56,038 INFO    MainThread:22428 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}

